### PR TITLE
Offer Slice card

### PR DESCRIPTION
**Pull request by igorp1**
__what__

Adds slice card component to our library  

Screenshot 2023-11-23 at 11 12 50 AM

### CODE REVIEW

File: src/components/OfferSlice/OfferSlice.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import { OfferSliceDetailItem } from "./OfferSliceDetailItem";
+import { VSpace } from "@components/shared/VSpace";
+
+export interface OfferSliceProps {
+  slice: OfferSliceType;
+  showFullDate?: boolean;
+  showFlightNumbers?: boolean;
+  hideFareBrand?: boolean;
+  highlightAll?: boolean;
+  // keysToHighlight?: string[];
```
Comment (andrejak) [inline src/components/OfferSlice/OfferSlice.tsx:20]: Do you think it's worth keeping these comments in or could we just add them as we need them later?

File: src/stories/OfferSlice.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  icao_code: "LFPG",
+  id: "air_paris_cdg",
+  latitude: 49.0079,
+  longitude: 2.5508,
+  name: "Paris Charles de Gaulle Airport",
+  time_zone: "Europe/Paris",
+};
+
+const MOCK_STOP: OfferSliceSegmentStop = {
+  // TODO(idp): type errors will go away once we update to the latest version of the API
```
Comment (andrejak) [inline src/stories/OfferSlice.stories.tsx:43]: Should we do that in this PR?

**Review by andrejak — APPROVED** [review]

File: src/stories/OfferSlice.stories.tsx (hunk 2 above)
Comment (igorp1) [reply src/stories/OfferSlice.stories.tsx]: Good point, done!

File: src/components/OfferSlice/OfferSlice.tsx (hunk 1 above)
Comment (igorp1) [reply src/components/OfferSlice/OfferSlice.tsx]: I'll remove it for now and like you said "add them as we need them later"

**Pull request merged**